gdmlGeneratorV1_materials.pl

```diff
@@ -830,7 +833,7 @@
 	    	<property name=\"REFLECTIVITY\" ref=\"Aluminium_Surf_Reflectivity\"/>
         </opticalsurface>";
 print def "
-        <opticalsurface name=\"Mylar\" model=\"glisur\" finish=\"polishlumirrorair\" type=\"dielectric_metal\" value=\"1.0\">
+        <opticalsurface name=\"Mylar\" model=\"glisur\" finish=\"polishedlumirrorair\" type=\"dielectric_metal\" value=\"1.0\">
```

Comment:
This was a legitimate bug, but I don't think the polish properties have a large impact on remoll outputs (which may be its own problem)